### PR TITLE
removed protocol-specific conformance statements

### PR DESCRIPTION
**Pull request by brentzundel**
@David-Chadwick I would like your feedback on the changes I'm recommending to your PR.

Signed-off-by: Brent <brent.zundel@gmail.com>

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-data-model/pull/449.html" title="Last updated on Mar 12, 2019, 5:25 PM UTC (db5d7f5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/449/9a486cc...brentzundel:db5d7f5.html" title="Last updated on Mar 12, 2019, 5:25 PM UTC (db5d7f5)">Diff</a>

### CODE REVIEW

**Comment by brentzundel** [comment]
This PR proposes changes to the 'chadwick-conformance' branch. 
If merged, it would only change that branch, not the 'gh-pages' branch.

File: index.html [hunk 1]
```diff
@@ -1158,16 +1162,18 @@ <h3>Credential Subject</h3>
...
 
         <dl>
           <dt>credentialSubject</dt>
           <dd>
-The value of the <code>credentialSubject</code> <a>property</a> MUST be
-an object that contains one or more properties that are related to the
-<a>subject</a> of the <a>verifiable credential</a>. The object MAY contain an
+The value of the <code>credentialSubject</code> <a>property</a> is defined as
+a set of objects that contain one or more properties that are each related to a
+<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
```
Comment (burnburn) [inline index.html]: I don't believe this is correct.  My suggestion: "The value of the <code>credentialSubject</code> <a>property</a> MUST be an object containing one or more properties.  This specification defines these properties to be related to the <a>subject</a> of the <a>verifiable credential</a>.  The object MAY"

Comment (msporny) [reply index.html]: Your suggestion would break the spec. :) -- `credentialSubject` can be an array, or more specifically, a set.

Comment (burnburn) [reply index.html]: Oops, forget what I said.

**Pull request merged**